packages/tradershub/src/features/cfd/modals/TopUpModal/TopUpModal.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { useIsEuRegion, useMT5Deposit, useOtherCFDPlatformsDeposit } from '@deriv/api';
+import { Button, Heading, Text, useBreakpoint } from '@deriv/quill-design';
+import { Modal } from '../../../../components/Modal';
+import { THooks, TPlatforms } from '../../../../types';
+import { CFDPlatforms, MarketType, MarketTypeDetails, PlatformDetails } from '../../constants';
+
+type TTopUpModalProps = {
+    account: THooks.CtraderAccountsList | THooks.DxtradeAccountsList | THooks.MT5AccountsList;
+    platform: TPlatforms.All;
+};
+
+const TopUpModal = ({ account, platform }: TTopUpModalProps) => {
+    const { isDesktop } = useBreakpoint();
+    const { data: isEuRegion } = useIsEuRegion();
+    const { mutateAsync: MT5Deposit } = useMT5Deposit();
+    const { mutateAsync: OtherCFDPlatformsDeposit } = useOtherCFDPlatformsDeposit();
+
+    const topUpVirtual = async () => {
+        if (platform === CFDPlatforms.MT5) {
+            await MT5Deposit({
+                to_mt5: account?.login ?? '',
+            });
+        } else {
+            await OtherCFDPlatformsDeposit({
+                platform,
+                to_account: (account as THooks.CtraderAccountsList | THooks.DxtradeAccountsList).account_id ?? '',
+            });
+        }
+    };
+
+    const platformTitle = PlatformDetails[platform].title;
+    const marketTypeDetails = MarketTypeDetails(isEuRegion)[account.market_type ?? MarketType.ALL];
+    const marketTypeTitle = marketTypeDetails?.title ?? '';
+    const title = platform === CFDPlatforms.MT5 ? `${platformTitle} ${marketTypeTitle}` : platformTitle;
+
+    const HeadingTag = isDesktop ? Heading.H3 : Heading.H2;
+    const balance =
+        platform === CFDPlatforms.CTRADER
+            ? (account as THooks.CtraderAccountsList)?.formatted_balance
+            : account?.display_balance;
+
+    return (
+        <Modal className='max-w-[330px] md:max-w-[440px]'>
```

Comment:
use `lg` or `sm` instead of `md` we have two different screen size either mobile or desktop nothing in between.
```suggestion
        <Modal className='max-w-[330px] lg:max-w-[440px]'>
```

Reply:
@aizad-deriv please, check the screenshot below from quill-design source code

<img width="552" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/83e0ed23-1ca2-449b-8e2e-b9bd494fb5a6">

You can see `isMobile` is below `md` (max-width is md) so when I use `md` here I use `min-width is md`, so it means more than mobile. Correct me if I'm wrong, but as we don't have tablet design and use mobile first approach then desktop view is width more than md.

Reply:
@sergei-deriv There isn't no difference for this is it's behaviour since we do not have tablet design, but to keep our consintancy among the code we can change it to `lg`

Reply:
@mahdiyeh-deriv but in this case if I'll use `lg` these styles will be applied when screen size is more than 1024 px (`lg`), but we are using `isMobile` from `useBreakpoint` from `quill-design` and it's true for width below than 768 px (`md`). In this case for width, for example, 800 px we will show the content for desktop (`!isMobile`), but styles will be for mobile (because we are using different breakpoints for js and css)... Hope you understand what I mean